src/code/z_sram_NES.c

```diff
@@ -1,67 +1,1661 @@
+        }
+
+        gSaveContext.save.playerData.newf[0] = 'Z';
+        gSaveContext.save.playerData.newf[1] = 'E';
+        gSaveContext.save.playerData.newf[2] = 'L';
+        gSaveContext.save.playerData.newf[3] = 'D';
+        gSaveContext.save.playerData.newf[4] = 'A';
+        gSaveContext.save.playerData.newf[5] = '3';
+
+        gSaveContext.save.checksum = Sram_CalcChecksum(&gSaveContext.save, sizeof(Save));
+
+        Lib_MemCpy(*sramCtx->saveBuf, &gSaveContext.save, sizeof(Save));
+        Lib_MemCpy(&(*sramCtx->saveBuf)[0x2000], &gSaveContext.save, sizeof(Save));
+
+        for (i = 0; i < ARRAY_COUNT(gSaveContext.save.playerData.newf); i++) {
+            fileChooseCtx->newf[fileChooseCtx->unk_24480][i] = gSaveContext.save.playerData.newf[i];
+        }
+
+        fileChooseCtx->unk_2440C[fileChooseCtx->unk_24480] = gSaveContext.save.playerData.deaths;
+
+        for (i = 0; i < ARRAY_COUNT(gSaveContext.save.playerData.playerName); i++) {
+            fileChooseCtx->unk_24414[fileChooseCtx->unk_24480][i] = gSaveContext.save.playerData.playerName[i];
+        }
+
+        fileChooseCtx->healthCapacity[fileChooseCtx->unk_24480] = gSaveContext.save.playerData.healthCapacity;
+        fileChooseCtx->health[fileChooseCtx->unk_24480] = gSaveContext.save.playerData.health;
+        fileChooseCtx->unk_24454[fileChooseCtx->unk_24480] = gSaveContext.save.inventory.dungeonKeys[9];
+        fileChooseCtx->unk_24444[fileChooseCtx->unk_24480] = gSaveContext.save.inventory.questItems;
+        fileChooseCtx->unk_24458[fileChooseCtx->unk_24480] = gSaveContext.save.time;
+        fileChooseCtx->unk_24460[fileChooseCtx->unk_24480] = gSaveContext.save.day;
+        fileChooseCtx->unk_24468[fileChooseCtx->unk_24480] = gSaveContext.save.isOwlSave;
+        fileChooseCtx->rupees[fileChooseCtx->unk_24480] = gSaveContext.save.playerData.rupees;
+        fileChooseCtx->unk_24474[fileChooseCtx->unk_24480] =
+            (gSaveContext.save.inventory.upgrades & gUpgradeMasks[4]) >> gUpgradeShifts[4];
+
+        for (i = 0, phi_a0 = 0; i < ARRAY_COUNT(gSaveContext.save.inventory.masks); i++) {
+            if (gSaveContext.save.inventory.masks[(s32)i] != ITEM_NONE) {
+                phi_a0++;
+            }
+        }
+
+        fileChooseCtx->unk_24478[fileChooseCtx->unk_24480] = phi_a0;
+        fileChooseCtx->unk_2447C[fileChooseCtx->unk_24480] =
+            (gSaveContext.save.inventory.questItems & 0xF0000000) >> 0x1C;
+    }
+    gSaveContext.save.time = D_801F6AF0;
+    gSaveContext.unk_3F3F = D_801F6AF2;
+}
+
+void func_80146DF8(SramContext* sramCtx) {
+    if (gSaveContext.unk_3F3F) {
+        gSaveContext.options.language = 1;
+        Lib_MemCpy(*sramCtx->saveBuf, &gSaveContext.options, sizeof(SaveOptions));
+    }
+}
+
+void func_80146E40(GameState* gameState, SramContext* sramCtx) {
+    if (&gSaveContext.save) {}
+
+    func_801A3D98(gSaveContext.options.audioSetting);
+}
```

Comment:
Wow, that's a small function to need this sort of thing. It's not a `((void)0,...)`, right?

Reply:
No, it isn't :s

Reply:
I'm even more confused by this because it doesn't even use `save` 🤔